src/pkg/provisioner/provisioner.go

```diff
@@ -45,11 +47,96 @@ func setup(dockerCredentialGCR string, systemd *systemdClient) error {
 	return nil
 }
 
-func cleanup(stateDir string) error {
+func stopServices(systemd *systemdClient) error {
+	log.Println("Stopping services...")
+	if err := systemd.stop("crash-reporter.service"); err != nil {
```

Comment:
nit: maintain a list of services to stop something like this `servicesToStop = [...]string{"service1", "service2", ...}` and loop to stop all of them.

Reply:
Done